### PR TITLE
add basic outline of questionnaire form

### PR DESCRIPTION
**Pull request by ThomasQuesadilla**
Added some jinja templating to condense selection html. Still working on applying CSS and more functionality (e.g. different questions based on selection of the previous screen).

Apologies for the long PR, the only new thing that was added from the PR in original repo is the countries/states parsing and templating so that no 196 line selection would exist in the html template.

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/QuestionnaireServlet.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      System.err.println("templates dir not found!");
+    }
+
+    Map<String, Object> context = new HashMap<>();
+    context.put("url", "/");
+    context.put("countries", getCountries());
+    context.put("states", getStates());
+    String template =
+        Resources.toString(
+            this.getClass().getResource("/templates/questionnaire.html"), Charsets.UTF_8);
```
Comment (sagunb) [inline src/main/java/com/google/sps/servlets/QuestionnaireServlet.java]: move all of these to the constants file

Comment (sagunb) [reply src/main/java/com/google/sps/servlets/QuestionnaireServlet.java]: these = hard coded resource locations

File: src/main/resources/templates/questionnaire.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    <h3> Step 2: More Personal Information </h3>
+    <p> This information is used to find a person that best fits you.
+       Feel free to fill in with whatever you're comfortable sharing: everything in this step is optional </p>
+    <div id="ethnicity">
+      <p> Check off whatever best applies to you </p>
+      <input type="checkbox" name="asian" value="asian">
+      <label for="asian"> Asian</label><br>
+      <input type="checkbox" name="black-african" value="black-african">
+      <label for="black-african"> Black/African</label><br>
+      <input type="checkbox" name="caucasian" value="caucasian">
```
Comment (sagunb) [inline src/main/resources/templates/questionnaire.html]: prefer to have these multiple choice options generated in the backend this way the list of valid options is always in sync with what the backend can accept.

**Review by sagunb — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/sps/data/TimeZoneInfo.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.TimeZone;
+
+public class TimeZoneInfo {
+
+  private static final double MILLISECONDS_TO_HOURS = 3600000.0;
```
Comment (linguistrix) [inline src/main/java/com/google/sps/data/TimeZoneInfo.java]: nit but better to name it something that makes it clear what direction the conversion is in.

MILLISECONDS_PER_HOUR for eg. makes it clear.

**Review by linguistrix — CHANGES_REQUESTED** [review]

**Review by sagunb — APPROVED** [review]

**Pull request merged**